src/modules/jcms_rest/src/Plugin/rest/resource/PersonListRestResource.php

```diff
@@ -102,18 +101,10 @@ protected function filterSubjects(QueryInterface &$query) {
   /**
    * Apply filter for type by amending query.
    */
-  protected function filterType(QueryInterface &$query) {
-    $type = $this->getRequestOption('type');
-    if (!is_null($type)) {
-      $entityManager = \Drupal::service('entity_field.manager');
-      $fields = $entityManager->getFieldStorageDefinitions('node', 'person');
-      $options = options_allowed_values($fields['field_person_type']);
-      if (!in_array($type, array_keys($options))) {
-        throw new JCMSBadRequestHttpException(t('Invalid type'));
-      }
-      else {
-        $query->condition('field_person_type.value', $type);
-      }
+  protected function filterTypes(QueryInterface &$query) {
+    $types = (array) $this->getRequestOption('type');
+    if (!empty($types)) {
+      $query->condition('field_person_type.value', $types, 'IN');
```

Comment:
Just want to confirm that this is supposed to be an `OR` query. I will just run this locally to confirm that if a person matches on multiple types that they don't appear twice in the listing. I agree we need better test coverage on this.

Reply:
`type` has a cardinality of 1 so there is no risk of a person appearing twice. This is an `OR` query. Happy to ship this.